packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -54,7 +90,7 @@ describe('Migration', () => {
       cy.findByText(defaultMessages.migration.wizard.step4.button).click()
       cy.findByText(defaultMessages.migration.wizard.step5.button).click()
 
-      cy.findByText(defaultMessages.setupWizard.selectFramework.description).should('be.visible')
+      // cy.findByText(defaultMessages.setupWizard.selectFramework.description).should('be.visible')
```

Comment:
@elevatebart is this working right now? It does nothing when clicked, also I think we agreed it shouldn't go to that page

Reply:
For now, it should still go to this page. If it was broken, that is a bug.

Reply:
let me see what's happening